src/main/java/uk/nhs/tis/trainee/notifications/event/LtftListener.java

```diff
@@ -53,6 +54,12 @@
 @Component
 public class LtftListener {
 
+  public static final Map<String, Set<NotificationType>> LTFT_UPDATE_EXPLICIT_NOTIFICATION_TYPES
```

Comment:
This was the way I thought of accommodating the fact that a single event can now trigger more than one notification, but it feels not very generic

Reply:
I'm leaning towards keeping it inline in the switch statement for readability

I know you're using the same map for the test expectations too, though there's an argument that we probably shouldn't. If the expectations of the test change automatically based on an implementation change we'd never be prompted to double check our work, as it wouldn't fail if we removed the wrong thing.

Reply:
The other option, which is more work but might be the "proper" way would be to have a new queue for TPD notifications, with a separate listener similar to the current one.

We could filter on the SNS subscription to just get the status updates that TPDs care about. Feels a bit less forced and removes the adding complexity of multiple/coupled notifications. As it stands we'll either have to write logic in to the `EmailService` to determine trainee vs TPD vs other, which feels super hacky. Or we'll branch in LtftListener to call a different method anyway, at that point we may as well have another listener.